### ISSUE TITLE
WRONG_INDENTATION warning message ignores `indentationSize` setting

### ISSUE THREAD

**Issue by colesnodgrass** [saveourtool/diktat#1886]
## Describe the bug
`WRONG_INDENTATION` message is hard-coded to `4 spaces` instead of being based on the `indentationSize` value.
https://github.com/saveourtool/diktat/blob/46ec9b912eb604fe411898e54ec65e61ea47fab1/diktat-rules/src/main/kotlin/com/saveourtool/diktat/ruleset/constants/Warnings.kt#L113

Given the following configuration
```yaml
- name: WRONG_INDENTATION
  enabled: true
  configuration:
    indentationSize: 2
```

## Expected behavior
```
[WRONG_INDENTATION] only spaces are allowed for indentation and each indentation should equal to 2 spaces (tabs are not allowed): expected 2 but was 4 (diktat-ruleset:indentation)
```

## Observed behavior
```
[WRONG_INDENTATION] only spaces are allowed for indentation and each indentation should equal to 4 spaces (tabs are not allowed): expected 2 but was 4 (diktat-ruleset:indentation)
```

## Environment information
* diktat version: 2.0.0
* build tool (maven/gradle): gradle
* how is diktat run (CLI, plugin, etc.): ./gradlew diktatCheck
* kotlin version: 1.9.21

**Comment by nulls** [comment]
Hi @colesnodgrass, thanks for raising the issue.

We will fix the warn message
